linkerd/stack/src/watch.rs

```diff
@@ -48,31 +43,45 @@ pub struct SpawnWatch<S> {
 // === impl NewSpawnWatch ===
 
 impl<P, N> NewSpawnWatch<P, N> {
-    pub fn new(new_update: N) -> Self {
+    pub fn new(inner: N) -> Self {
         Self {
-            new_update,
+            inner,
             _marker: std::marker::PhantomData,
         }
     }
+
+    pub fn layer() -> impl tower::layer::Layer<N, Service = Self> + Clone {
+        crate::layer::mk(Self::new)
+    }
+
+    pub fn layer_into<T>(
```

Comment:
maybe `layer_from_tuple` or something because the name of the inner service changed?

Reply:
i think _into_ reads more clearly here